app/build.gradle

```diff
@@ -134,22 +133,21 @@ dependencies {
     // Needed for locating the user -  todo - do it with here positioning
     implementation 'com.google.android.gms:play-services-location:18.0.0'
 
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
I've removed the explicit dependency on the Kotlin standard library since [it's added automatically starting with Kotlin 1.4](https://stackoverflow.com/a/64988522/5723188).